go.mod

```diff
@@ -25,6 +25,7 @@ require (
 
 require (
 	github.com/cenkalti/backoff/v4 v4.3.0
+	github.com/cenkalti/backoff/v5 v5.0.2
```

Comment:
why do we have both v4 and v5?

Reply:
v4 is used somewhere else in a test script. I would want to use the latest version and maybe in a separate PR remove/change the test script